docs/en/pid-eaa-issuance.rst

```diff
@@ -364,12 +378,12 @@ The requests to the PID/(Q)EAA authorization endpoint MUST use the HTTP POST met
     * - **request**
       - It MUST be a signed JWT. The private key corresponding to the public one in the ``cnf`` parameter inside the Wallet Attestation MUST be used for signing the Request Object.
       - `OpenID Connect Core. Section 6 <https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests>`_
-    * - **client_assertion_type**
-      - It MUST be set to ``urn:ietf:params:oauth:client-assertion-type:jwt-client-attestation``.
-      - `oauth-attestation-draft <https://vcstuff.github.io/draft-ietf-oauth-attestation-based-client-auth/draft-ietf-oauth-attestation-based-client-auth.html>`_.
-    * - **client_assertion**
-      - It MUST be set to a value containing the Wallet Attestation JWT and the Proof of Possession, separated with the ``~`` character.
-      - `oauth-attestation-draft <https://vcstuff.github.io/draft-ietf-oauth-attestation-based-client-auth/draft-ietf-oauth-attestation-based-client-auth.html>`_.
+    * - **OAuth-Client-Attestation**
+      - It MUST be set to a value containing the Wallet Attestation JWT.
+      - `OAUTH-ATTESTATION-CLIENT-AUTH`_ draft 03.
+    * - **OAuth-Client-Attestation-PoP**
+      - It MUST be set to a value containing the Wallet Attestation JWT Proof of Possession.
+      - `OAUTH-ATTESTATION-CLIENT-AUTH`_ draft 03.
```

Comment:
```suggestion
```

Reply:
These parameters must be included in the HTTP header request and not in the payload of the request object.